app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,42 +33,103 @@ export default function NoteTable({
-
-                    if (index > -1) {
-                      setSelectedNoteIds([
-                        ...selectedNoteIds.slice(0, index),
-                        ...selectedNoteIds.slice(index + 1),
-                      ]);
-                    } else {
-                      setSelectedNoteIds([...selectedNoteIds, note.id]);
-                    }
-                  }}
-                />
-              </td>
-              <td>
-                <div className="flex flex-row items-center justify-start text-gray-900">
-                  {note.title}
-                </div>
-              </td>
-              <td>{note.description}</td>
-            </tr>
-          ))}
+          {notes.map(note => {
+            const contact = {
+              name: "Oliver Smith",
+            };
+            const createdDate = format(new Date(note.created_at), "MMM, d y");
+            const dueDate = note?.due_date
+              ? format(new Date(note.due_date), "MMM, d y")
+              : "--";
+            const badgeColor =
+              issueTypeBadgeColors[notes.issueType || "internal"];
+            return (
+              <tr
+                key={note.id}
+                className={"group cursor-pointer bg-white hover:bg-gray-50"}
+              >
+                <td>
+                  <Checkbox
+                    checked={selectedNoteIds.includes(note.id)}
+                    onClick={event => {
+                      event.stopPropagation();
+                      const index = selectedNoteIds.indexOf(note.id);
+                      if (index > -1) {
+                        setSelectedNoteIds([
+                          ...selectedNoteIds.slice(0, index),
+                          ...selectedNoteIds.slice(index + 1),
+                        ]);
+                      } else {
+                        setSelectedNoteIds([...selectedNoteIds, note.id]);
+                      }
+                    }}
+                  />
+                </td>
+                <td>
+                  <div className="flex flex-row items-center justify-start text-gray-900">
+                    <a className="text-tableContent">{note.title}</a>
+                  </div>
+                </td>
+                <td className="truncate w-1/5 text-tableContent text-coolGray-700">
+                  {note.description}
+                </td>
+                <td className="text-tableContent text-center text-coolGray-700">
+                  <Badge color={badgeColor}>Internal</Badge>
+                </td>
+                <td className="text-tableContent text-center text-coolGray-700">
+                  {createdDate}
+                </td>
+                <td className="text-tableContent text-center text-coolGray-700">
+                  {dueDate}
+                </td>
+                <td>
+                  <Avatar className="my-0 mx-auto" contact={contact} />
+                </td>
+                <td className="hidden group-hover:flex">
```

Comment:
We don't need to add these classnames now since we are adding class `nui-table--actions`.

Reply:
Resolved in #9